exasol_sagemaker_extension/autopilot_prediction.py

```diff
@@ -24,7 +24,9 @@ def run(self, ctx):
 
         if endpoint_info_json['status'] == 'deployed':
             data_df = ctx.get_dataframe()
-            pred = self.prediction_method(endpoint_info_json['name'], data_df)
+            pred = self.prediction_method(
+                endpoint_info_json['endpoint_name'].upper(), data_df)
```

Comment:
@tkilias 
As far as I can see, we can only use csvserialization when sending data to the endpoint. As a result, the prediction results come as strings. We need to cast the prediction result to the data type we will emit in the UDF. 

I couldn't get data type in UDF (via output_columns). How can I get this? The last option is the model_connection string in which I will add as hard-coded one.